Adding PER and getMaxValueIndex function

raimannma opened the pull request:
Adding PER, but it's not very efficient right now.
Adding the simple function getMaxValueIndex

christianechevarria commented:
@raimannma thanks for adding this! Usually we'd add tests before merging, but this branch is super experimental so we'll just go for it lol 

By the way, did you find that the RL agent was able to learn?

christianechevarria commented:
@all-contributors please add @raimannma for code

raimannma commented:
> @raimannma thanks for adding this! Usually we'd add tests before merging, but this branch is super experimental so we'll just go for it lol
> 
> By the way, did you find that the RL agent was able to learn?

I haven't done any tests, so i can't tell you anything about the RL Agent.

But as you saw i rewrote neataptic to Java and in my java port i already implemented a reinforcement learning agent.
